test/integration/test_grpc_service.py

```diff
@@ -2,7 +2,8 @@
 import pytest
 
 from test.integration.integration_test_fixtures import http_test_server_fixture
-from test.integration.utility import *
+from test.integration.utility import (isSanitizerRun, assertEqual, assertIn, assertGreaterEqual,
```

Comment:
Looks like this now imports functions from the utility module. What we really should be importing is the utility module itself.

from test.integration import utility

And then calls will look like.
utility.assertEqual

Ideally utility would have a better more descriptive name, but we can address that separately. The Python style we are following internally suggests to only import packages or modules, that way it is easy to say where some function is defined. If the reader just sees "assertEqual", they have to do some digging.

Reply:
That makes sense, in the benchmark PR I noticed pep8 had trouble with the * imports too.
Changed what was relevant to this PR in 66f658553658b31ad93fac5f1dfc4b7871045265
Filed https://github.com/envoyproxy/nighthawk/issues/371 earlier to track doing a pass over
the all the other code, to change the rest.

Reply:
Thank you for filing the issue.